model/item.h

```diff
@@ -65,8 +65,8 @@ namespace minsky
   /// bounding box information (at zoom=1 scale)
   class BoundingBox
   {
-    float left=0, right=0, top, bottom;
   public:
+    float left=0, right=0, top, bottom;  
```

Comment:
These fields really should remain private, and only updated via the update() call, which uses cairo_ink_extents.

My thought is Item should sport two new member fields (m_width, m_height - say), and these are used to control how large the icon is on campus.